src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformConfigFormHelper.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+import * as Yup from 'yup'
+import { Connectors } from '@connectors/constants'
+import type { ConnectorInfoDTO } from 'services/cd-ng'
+import type { StringKeys } from 'framework/strings'
+import {
+  buildBitbucketPayload,
+  buildGithubPayload,
+  buildGitlabPayload,
+  buildGitPayload
+} from '@connectors/pages/connectors/utils/ConnectorUtils'
+
+export const AllowedTypes = ['Git', 'Github', 'GitLab', 'Bitbucket']
+export type ConnectorTypes = 'Git' | 'Github' | 'GitLab' | 'Bitbucket'
+
+export const tfVarIcons: any = {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232256)
```suggestion
export const tfVarIcons: unknown = {
```